apps/client/src/components/user-toasts/user-toasts.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Dialog,
+  DialogTitle,
+  createTheme,
+  DialogContent,
+  ThemeProvider,
+} from '@mui/material';
+import { useEffect, useState } from 'react';
+import {
+  useLoginMutation,
+  useSignUpMutation,
+  useGetAllToastsByUserQuery,
+} from '../../store';
+import { Card } from '../card';
+import { ToastsCard } from '../toasts-card';
+
+type Props = {
+  option: string;
+};
+
+export const DisplayToasts: React.FC<Props> = ({ option }) => {
+  const [dialogOpen, setDialogOpen] = useState<boolean>(false);
+  useEffect(() => {
+    setDialogOpen(option === 'השתיות שלי');
+  }, [option]);
+  const [, { data: signUpData }] = useSignUpMutation({
+    fixedCacheKey: 'signupResult',
+  });
+
+  const [, { data: signInData }] = useLoginMutation({
+    fixedCacheKey: 'signinResult',
+  });
+  const id = signUpData?.id || signInData?.id;
+  const { data: allToasts } = useGetAllToastsByUserQuery(id ? id : '');
+  const darkTheme = createTheme({
+    palette: {
+      mode: 'dark',
+    },
+  });
+  const handleClickAway = () => {
+    setDialogOpen(false);
+  };
+  const dialogStyle = {
```

Comment:
I can't see it from here but make sure that you're not defining another `dialogStyle` in another file with the same attributes. if so that would be code duplication